lbr_examples/scripts/lbr_ext_torque_example.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/python3
+import rclpy
+from rclpy import qos
+from rclpy.node import Node
+import optas
+
+import numpy as np
+
+from lbr_fri_msgs.msg import LBRState
+
+from geometry_msgs.msg import Wrench, Point
+from visualization_msgs.msg import Marker
+
+
+class ExtTorqueExampleNode(Node):
+    def __init__(self):
+        super().__init__("lbr_ext_torque_example")
+        self.declare_parameter("lbr_description")
+        ee_link = "storz_tilt_endoscope_link_cm_optical"
```

Comment:
storz optical is not part of this package. Please define ee_link as ros parameter and use a link that actually exists in the stack